lib/nunjucks.js

```diff
@@ -28,7 +28,7 @@ module.exports = (eleventyConfig) => {
 
   const searchPaths = [
     './node_modules/@x-govuk/govuk-eleventy-plugin',
-    resolveNpmModule('govuk-frontend'),
+    './node_modules/govuk-frontend/dist',
```

Comment:
This removes the use of `resolveModule` as not quite sure how I can make this work with a sub-directory.